test/atp/testdrive/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Copyright Materialize, Inc. All rights reserved.
```

Comment:
You may wish to remove ```test/testdrive/Dockerfile``` from the repository then.

Reply:
Righto! Fixed. Thanks.